### DISCUSSION TITLE
How to view the grid?

### DISCUSSION THREAD

**Discussion by CaneRandagio1983** [freeplane/freeplane#1309] [Help]
I went through all the menu commands, but I couldn't find the command to display the grid, 
I only found the option to change the size of the grid.

How the hell do I view the grid?

Thank you

**Comment by macmarrum** [comment]
The grid is invisible.

**Comment by CaneRandagio1983** [comment]
> The grid is invisible.
Is it possible to display guidelines while moving the nodes, to be precise on alignment?
